### ISSUE TITLE
perlfaq4: the section on indented here documents should mention the ~ modifier

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#16799]
Migrated from [rt.perl.org#133746](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=133746) (status was 'rejected')

Searchable as RT133746$

**Comment by p5pRT** [comment]
### From @jmdh


This is a bug report for perl from dom@&#8203;earth\.li\,
generated with the help of perlbug 1\.40 running under perl 5\.24\.1\.




---
As reported at http&#8203;://bugs\.debian\.org/915366&#8203;:

"perlfaq4 has a question "Why don't my \<\<HERE documents work?"\, and much
of the answer deals with ways to write indented here documents\. These
are largely kludges\, however\, and the faq should really mention the ~
modifier \(desribed in perlop\)\, which was introduced precisely in order
to provide a clean way to do this\."

The author suggests an addendum&#8203;:

"Beginning with Perl version 5\.26\, a much simpler and cleaner way to
write indented here documents has been added to the language&#8203;: the tilde
\(~\) modifier\. See "Indented Here\-docs" in perlop for details\."



---


---
Flags&#8203;:
&nbsp;    category=docs
&nbsp;    severity=low


---
Site configuration information for perl 5\.24\.1&#8203;:

Configured by Debian Project at Thu Nov 29 11&#8203;:11&#8203;:57 UTC 2018\.

Summary of my perl5 \(revision 5 version 24 subversion 1\) configuration&#8203;:
&nbsp;   
&nbsp;  Platform&#8203;:
&nbsp;    osname=linux\, osvers=3\.16\.0\, archname=x86\_64\-linux\-gnu\-thread\-multi
&nbsp;    uname='linux localhost 3\.16\.0 \#1 smp debian 3\.16\.0 x86\_64 gnulinux '
&nbsp;    config\_args='\-Dusethreads \-Duselargefiles \-Dcc=x86\_64\-linux\-gnu\-gcc \-Dcpp=x86\_64\-linux\-gnu\-cpp \-Dld=x86\_64\-linux\-gnu\-gcc \-Dccflags=\-DDEBIAN \-Wdate\-time \-D\_FORTIFY\_SOURCE=2 \-g \-O2 \-fdebug\-prefix\-map=/build/perl\-CWhbRh/perl\-5\.24\.1=\. \-fstack\-protector\-strong \-Wformat \-Werror=format\-security \-Dldflags= \-Wl\,\-z\,relro \-Dlddlflags=\-shared \-Wl\,\-z\,relro \-Dcccdlflags=\-fPIC \-Darchname=x86\_64\-linux\-gnu \-Dprefix=/usr \-Dprivlib=/usr/share/perl/5\.24 \-Darchlib=/usr/lib/x86\_64\-linux\-gnu/perl/5\.24 \-Dvendorprefix=/usr \-Dvendorlib=/usr/share/perl5 \-Dvendorarch=/usr/lib/x86\_64\-linux\-gnu/perl5/5\.24 \-Dsiteprefix=/usr/local \-Dsitelib=/usr/local/share/perl/5\.24\.1 \-Dsitearch=/usr/local/lib/x86\_64\-linux\-gnu/perl/5\.24\.1 \-Dman1dir=/usr/share/man/man1 \-Dman3dir=/usr/share/man/man3 \-Dsiteman1dir=/usr/local/man/man1 \-Dsiteman3dir=/usr/local/man/man3 \-Dusesitecustomize \-Duse64bitint \-Dman1ext=1 \-Dman3ext=3perl
\-Dpager=/usr/bin/sensible\-pager \-Uafs \-Ud\_csh \-Ud\_ualarm \-Uusesfio \-Uusenm \-Ui\_libutil \-Uversiononly \-DDEBUGGING=\-g \-Doptimize=\-O2 \-dEs \-Duseshrplib \-Dlibperl=libperl\.so\.5\.24\.1'
&nbsp;    hint=recommended\, useposix=true\, d\_sigaction=define
&nbsp;    useithreads=define\, usemultiplicity=define
&nbsp;    use64bitint=define\, use64bitall=define\, uselongdouble=undef
&nbsp;    usemymalloc=n\, bincompat5005=undef
&nbsp;  Compiler&#8203;:
&nbsp;    cc='x86\_64\-linux\-gnu\-gcc'\, ccflags ='\-D\_REENTRANT \-D\_GNU\_SOURCE \-DDEBIAN \-fwrapv \-fno\-strict\-aliasing \-pipe \-I/usr/local/include \-D\_LARGEFILE\_SOURCE \-D\_FILE\_OFFSET\_BITS=64'\,
&nbsp;    optimize='\-O2 \-g'\,
&nbsp;    cppflags='\-D\_REENTRANT \-D\_GNU\_SOURCE \-DDEBIAN \-fwrapv \-fno\-strict\-aliasing \-pipe \-I/usr/local/include'
&nbsp;    ccversion=''\, gccversion='6\.3\.0 20170516'\, gccosandvers=''
&nbsp;    intsize=4\, longsize=8\, ptrsize=8\, doublesize=8\, byteorder=12345678\, doublekind=3
&nbsp;    d\_longlong=define\, longlongsize=8\, d\_longdbl=define\, longdblsize=16\, longdblkind=3
&nbsp;    ivtype='long'\, ivsize=8\, nvtype='double'\, nvsize=8\, Off\_t='off\_t'\, lseeksize=8
&nbsp;    alignbytes=8\, prototype=define
&nbsp;  Linker and Libraries&#8203;:
&nbsp;    ld='x86\_64\-linux\-gnu\-gcc'\, ldflags =' \-fstack\-protector\-strong \-L/usr/local/lib'
&nbsp;    libpth=/usr/local/lib /usr/lib/gcc/x86\_64\-linux\-gnu/6/include\-fixed /usr/include/x86\_64\-linux\-gnu /usr/lib /lib/x86\_64\-linux\-gnu /lib/\.\./lib /usr/lib/x86\_64\-linux\-gnu /usr/lib/\.\./lib /lib
&nbsp;    libs=\-lgdbm \-lgdbm\_compat \-ldb \-ldl \-lm \-lpthread \-lc \-lcrypt
&nbsp;    perllibs=\-ldl \-lm \-lpthread \-lc \-lcrypt
&nbsp;    libc=libc\-2\.24\.so\, so=so\, useshrplib=true\, libperl=libperl\.so\.5\.24
&nbsp;    gnulibc\_version='2\.24'
&nbsp;  Dynamic Linking&#8203;:
&nbsp;    dlsrc=dl\_dlopen\.xs\, dlext=so\, d\_dlsymun=undef\, ccdlflags='\-Wl\,\-E'
&nbsp;    cccdlflags='\-fPIC'\, lddlflags='\-shared \-L/usr/local/lib \-fstack\-protector\-strong'

Locally applied patches&#8203;:
&nbsp;    DEBPKG&#8203;:debian/cpan\_definstalldirs \- Provide a sensible INSTALLDIRS default for modules installed from CPAN\.
&nbsp;    DEBPKG&#8203;:debian/db\_file\_ver \- https://bugs.debian.org/340047 Remove overly restrictive DB\_File version check\.
&nbsp;    DEBPKG&#8203;:debian/doc\_info \- Replace generic man\(1\) instructions with Debian\-specific information\.
&nbsp;    DEBPKG&#8203;:debian/enc2xs\_inc \- https://bugs.debian.org/290336 Tweak enc2xs to follow symlinks and ignore missing @&#8203;INC directories\.
&nbsp;    DEBPKG&#8203;:debian/errno\_ver \- https://bugs.debian.org/343351 Remove Errno version check due to upgrade problems with long\-running processes\.
&nbsp;    DEBPKG&#8203;:debian/libperl\_embed\_doc \- https://bugs.debian.org/186778 Note that libperl\-dev package is required for embedded linking
&nbsp;    DEBPKG&#8203;:fixes/respect\_umask \- Respect umask during installation
&nbsp;    DEBPKG&#8203;:debian/writable\_site\_dirs \- Set umask approproately for site install directories
&nbsp;    DEBPKG&#8203;:debian/extutils\_set\_libperl\_path \- EU&#8203;:MM&#8203;: set location of libperl\.a under /usr/lib
&nbsp;    DEBPKG&#8203;:debian/no\_packlist\_perllocal \- Don't install \.packlist or perllocal\.pod for perl or vendor
&nbsp;    DEBPKG&#8203;:debian/fakeroot \- Postpone LD\_LIBRARY\_PATH evaluation to the binary targets\.
&nbsp;    DEBPKG&#8203;:debian/instmodsh\_doc \- Debian policy doesn't install \.packlist files for core or vendor\.
&nbsp;    DEBPKG&#8203;:debian/ld\_run\_path \- Remove standard libs from LD\_RUN\_PATH as per Debian policy\.
&nbsp;    DEBPKG&#8203;:debian/libnet\_config\_path \- Set location of libnet\.cfg to /etc/perl/Net as /usr may not be writable\.
&nbsp;    DEBPKG&#8203;:debian/mod\_paths \- Tweak @&#8203;INC ordering for Debian
&nbsp;    DEBPKG&#8203;:debian/prune\_libs \- https://bugs.debian.org/128355 Prune the list of libraries wanted to what we actually need\.
&nbsp;    DEBPKG&#8203;:fixes/net\_smtp\_docs \- \[rt\.cpan\.org \#36038\] https://bugs.debian.org/100195 Document the Net&#8203;::SMTP 'Port' option
&nbsp;    DEBPKG&#8203;:debian/perlivp \- https://bugs.debian.org/510895 Make perlivp skip include directories in /usr/local
&nbsp;    DEBPKG&#8203;:debian/deprecate\-with\-apt \- https://bugs.debian.org/747628 Point users to Debian packages of deprecated core modules
&nbsp;    DEBPKG&#8203;:debian/squelch\-locale\-warnings \- https://bugs.debian.org/508764 Squelch locale warnings in Debian package maintainer scripts
&nbsp;    DEBPKG&#8203;:debian/skip\-upstream\-git\-tests \- Skip tests specific to the upstream Git repository
&nbsp;    DEBPKG&#8203;:debian/patchlevel \- https://bugs.debian.org/567489 List packaged patches for 5\.24\.1\-3\+deb9u5 in patchlevel\.h
&nbsp;    DEBPKG&#8203;:debian/skip\-kfreebsd\-crash \- https://bugs.debian.org/628493 \[perl \#96272\] Skip a crashing test case in t/op/threads\.t on GNU/kFreeBSD
&nbsp;    DEBPKG&#8203;:fixes/document\_makemaker\_ccflags \- https://bugs.debian.org/628522 \[rt\.cpan\.org \#68613\] Document that CCFLAGS should include $Config\{ccflags\}
&nbsp;    DEBPKG&#8203;:debian/find\_html2text \- https://bugs.debian.org/640479 Configure CPAN&#8203;::Distribution with correct name of html2text
&nbsp;    DEBPKG&#8203;:debian/perl5db\-x\-terminal\-emulator\.patch \- https://bugs.debian.org/668490 Invoke x\-terminal\-emulator rather than xterm in perl5db\.pl
&nbsp;    DEBPKG&#8203;:debian/cpan\-missing\-site\-dirs \- https://bugs.debian.org/688842 Fix CPAN&#8203;::FirstTime defaults with nonexisting site dirs if a parent is writable
&nbsp;    DEBPKG&#8203;:fixes/memoize\_storable\_nstore \- \[rt\.cpan\.org \#77790\] https://bugs.debian.org/587650 Memoize&#8203;::Storable&#8203;: respect 'nstore' option not respected
&nbsp;    DEBPKG&#8203;:debian/regen\-skip \- Skip a regeneration check in unrelated git repositories
&nbsp;    DEBPKG&#8203;:debian/makemaker\-pasthru \- https://bugs.debian.org/758471 Pass LD settings through to subdirectories
&nbsp;    DEBPKG&#8203;:debian/makemaker\-manext \- https://bugs.debian.org/247370 Make EU&#8203;::MakeMaker honour MANnEXT settings in generated manpage headers
&nbsp;    DEBPKG&#8203;:debian/devel\-ppport\-reproducibility \- https://bugs.debian.org/801523 Sort the list of XS code files when generating RealPPPort\.xs
&nbsp;    DEBPKG&#8203;:debian/encode\-unicode\-bom\-doc \- https://bugs.debian.org/798727 Document Debian backport of Encode&#8203;::Unicode fix
&nbsp;    DEBPKG&#8203;:debian/kfreebsd\-softupdates \- https://bugs.debian.org/796798 Work around Debian Bug\#796798
&nbsp;    DEBPKG&#8203;:fixes/autodie\-scope \- https://bugs.debian.org/798096 Fix a scoping issue with "no autodie" and the "system" sub
&nbsp;    DEBPKG&#8203;:fixes/crosscompile\-no\-targethost \- \[23695c0\] \[perl \#127234\] Fix the Configure escape with usecrosscompile but no targethost
&nbsp;    DEBPKG&#8203;:fixes/memoize\-pod \- \[rt\.cpan\.org \#89441\] Fix POD errors in Memoize
&nbsp;    DEBPKG&#8203;:fixes/ok\-pod \- Added encoding for pod\.
&nbsp;    DEBPKG&#8203;:debian/hurd\-softupdates \- https://bugs.debian.org/822735 Fix t/op/stat\.t failures on hurd
&nbsp;    DEBPKG&#8203;:fixes/nntp\_docs \- https://bugs.debian.org/51962 Net&#8203;::NNTP&#8203;: Correct innd/nnrpd confusion in relation to Reader option
&nbsp;    DEBPKG&#8203;:fixes/math\_complex\_doc\_great\_circle \- https://bugs.debian.org/697567 \[rt\.cpan\.org \#114104\] Math&#8203;::Trig&#8203;: clarify definition of great\_circle\_midpoint
&nbsp;    DEBPKG&#8203;:fixes/math\_complex\_doc\_see\_also \- https://bugs.debian.org/697568 \[rt\.cpan\.org \#114105\] Math&#8203;::Trig&#8203;: add missing SEE ALSO
&nbsp;    DEBPKG&#8203;:fixes/math\_complex\_doc\_angle\_units \- https://bugs.debian.org/731505 \[rt\.cpan\.org \#114106\] Math&#8203;::Trig&#8203;: document angle units
&nbsp;    DEBPKG&#8203;:fixes/cpan\_web\_link \- https://bugs.debian.org/367291 CPAN&#8203;: Add link to main CPAN web site
&nbsp;    DEBPKG&#8203;:fixes/time\_piece\_doc \- https://bugs.debian.org/817925 Time&#8203;::Piece&#8203;: Improve documentation for add\_months and add\_years
&nbsp;    DEBPKG&#8203;:fixes/perlbug\-refactor \- https://bugs.debian.org/822463 \[perl \#128020\] perlbug&#8203;: Refactor duplicated file reading code
&nbsp;    DEBPKG&#8203;:fixes/perlbug\-linewrap \- https://bugs.debian.org/822463 \[perl \#128020\] perlbug&#8203;: wrap overly long lines
&nbsp;    DEBPKG&#8203;:fixes/hurd\_sigaction \- https://bugs.debian.org/825016 \[d54f4ed\] ext/POSIX/t/sigaction\.t&#8203;: Skip uid and pid tests on GNU/Hurd
&nbsp;    DEBPKG&#8203;:fixes/hurd\_hints \- \[4694301\] https://bugs.debian.org/825020 \[perl \#128279\] Modify hints for Hurd per Debian ticket 825020\.
&nbsp;    DEBPKG&#8203;:fixes/extutils\-parsexs\-reproducibility \- \[perl \#128517\] https://bugs.debian.org/829296 Make the output of ExtUtils&#8203;::ParseXS reproducible
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/sitecustomize\-in\-etc \- Look for sitecustomize\.pl in /etc/perl rather than sitelib on Debian systems
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/test\-suite\-without\-dot \- \[perl \#127810\] Patch unit tests to explicitly insert "\." into @&#8203;INC when needed\.
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/eumm\-without\-dot \- \[perl \#127810\] Add PERL\_USE\_UNSAFE\_INC support to EU&#8203;::MM for fortify\_inc support\.
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/cpan\-without\-dot \- \[perl \#127810\] Set PERL\_USE\_UNSAFE\_INC for cpan usage
&nbsp;    DEBPKG&#8203;:debian/document\_inc\_removal \- Document in perlvar that we remove '\.' from @&#8203;INC by default
&nbsp;    DEBPKG&#8203;:fixes/extutils\_makemaker\_reproducible \- https&#8203;://bugs\.debian\.org/835815 https://bugs.debian.org/834190 Make perllocal\.pod files reproducible
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/remove\-inc\-test \- Remove test for '\.' in @&#8203;INC as it might not be
&nbsp;    DEBPKG&#8203;:fixes/file\_path\_hurd\_errno \- File\-Path&#8203;: Fix test failure in Hurd due to hard\-coded ENOENT
&nbsp;    DEBPKG&#8203;:debian/hppa\_op\_optimize\_workaround \- https://bugs.debian.org/838613 Temporarily lower the optimization of op\.c on hppa due to gcc\-6 problems
&nbsp;    DEBPKG&#8203;:fixes/test\-builder\-warning \- https://bugs.debian.org/840968 Silence a 'used only once' warning in Test&#8203;::Builder
&nbsp;    DEBPKG&#8203;:fixes/longdblinf\-randomness \- \[dd68853\] \[perl \#130133\] https://bugs.debian.org/844752 Configure&#8203;: fix garbage filtering with 80\-bit long doubles
&nbsp;    DEBPKG&#8203;:debian/installman\-utf8 \- https://bugs.debian.org/840211 Generate man pages with UTF\-8 characters
&nbsp;    DEBPKG&#8203;:fixes/list\_assign\_leak \- \[1050723\] \[perl \#130766\] https://bugs.debian.org/855064 avoid a leak in list assign from/to magic values
&nbsp;    DEBPKG&#8203;:fixes/perlfunc\_inc\_doc \- \[a03e9f8\] https://bugs.debian.org/839536 \[perl \#130832\] Documentation fixes for '\.' possibly no longer being in @&#8203;INC
&nbsp;    DEBPKG&#8203;:fixes/file\_path\_chmod\_race \- https://bugs.debian.org/863870 \[rt\.cpan\.org \#121951\] Prevent directory chmod race attack\.
&nbsp;    DEBPKG&#8203;:fixes/extutils\_file\_path\_compat \- Correct the order of tests of chmod\(\)\. \(\#294\)
&nbsp;    DEBPKG&#8203;:debian/customized \- Update customized\.dat for files patched in Debian
&nbsp;    DEBPKG&#8203;:fixes/getopt\-long\-1 \- https://bugs.debian.org/855532 \[rt\.cpan\.org \#114999\] Fix bug RT\#114999
&nbsp;    DEBPKG&#8203;:fixes/getopt\-long\-2 \- \[rt\.cpan\.org \#120300\] Withdraw part of commit 5d9947fb445327c7299d8beb009d609bc70066c0\, which tries to implement more GNU getopt\_long campatibility\. GNU
&nbsp;    DEBPKG&#8203;:fixes/getopt\-long\-3 \- provide a default value for optional arguments
&nbsp;    DEBPKG&#8203;:fixes/getopt\-long\-4 \- https://bugs.debian.org/864544 \[rt\.cpan\.org \#122068\] Fix issue \#122068\.
&nbsp;    DEBPKG&#8203;:fixes/fbm\-instr\-crash \- \[bb152a4\] \[perl \#131575\] https://bugs.debian.org/864782 don't call Perl\_fbm\_instr\(\) with negative length
&nbsp;    DEBPKG&#8203;:debian/CVE\-2016\-1238/base\-pm\-amends\-pt2 \- \[1afa289\] Limit dotless\-INC effect on base\.pm with guard&#8203;:
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2017\-12837 \- https://bugs.debian.org/875596 \[perl \#131582\] \[f7e5417\] regcomp \[perl \#131582\]
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2017\-12883 \- https://bugs.debian.org/875597 \[perl \#131598\] \[40b3cda\] PATCH&#8203;: \[perl \#131598\]
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-6797 \- \[perl \#132227\] \(perl \#132227\) restart a node if we change to uni rules within the node and encounter a sharp S
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-6798/pt1 \- \[perl \#132063\] Heap buffer overflow
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-6798/pt2 \- \[perl \#132063\] v5\.24\.3&#8203;: fix TRIE\_READ\_CHAR and DECL\_TRIE\_TYPE to account for non\-utf8 target
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-6798/pt3 \- \[perl \#132063\] \(perl \#132063\) we should no longer warn for this code
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-6913 \- \[perl \#131844\] \(perl \#131844\) fix various space calculation issues in pp\_pack\.c
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-12015\-Archive\-Tar\-directory\-traversal \- https://bugs.debian.org/900834 \[rt\.cpan\.org \#125523\] Remove existing files before overwriting them
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-18311 \- Perl\_my\_setenv\(\); handle integer wrap
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-18312 \- for 5\.26 maint
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-18313 \- regcomp\.c&#8203;: Convert some strchr to memchr
&nbsp;    DEBPKG&#8203;:fixes/CVE\-2018\-18314 \- fix \#131649 \- extended charclass can trigger assert



---
@&#8203;INC for perl 5\.24\.1&#8203;:
&nbsp;    /etc/perl
&nbsp;    /usr/local/lib/x86\_64\-linux\-gnu/perl/5\.24\.1
&nbsp;    /usr/local/share/perl/5\.24\.1
&nbsp;    /usr/lib/x86\_64\-linux\-gnu/perl5/5\.24
&nbsp;    /usr/share/perl5
&nbsp;    /usr/lib/x86\_64\-linux\-gnu/perl/5\.24
&nbsp;    /usr/share/perl/5\.24
&nbsp;    /usr/local/lib/site\_perl
&nbsp;    /usr/lib/x86\_64\-linux\-gnu/perl\-base



---
Environment for perl 5\.24\.1&#8203;:
&nbsp;    HOME=/home/dom
&nbsp;    LANG=en\_GB\.UTF\-8
&nbsp;    LANGUAGE=en\_GB&#8203;:en
&nbsp;    LD\_LIBRARY\_PATH \(unset\)
&nbsp;    LOGDIR \(unset\)
&nbsp;    PATH=/home/dom/bin&#8203;:/usr/local/bin&#8203;:/usr/bin&#8203;:/bin&#8203;:/usr/local/games&#8203;:/usr/games&#8203;:/sbin&#8203;:/usr/sbin
&nbsp;    PERL\_BADLANG \(unset\)
&nbsp;    SHELL=/bin/bash

**Comment by p5pRT** [comment]
### From @jkeenan

On Tue\, 01 Jan 2019 13&#8203;:07&#8203;:38 GMT\, dom wrote&#8203;:
> 
> This is a bug report for perl from dom@&#8203;earth\.li\,
> generated with the help of perlbug 1\.40 running under perl 5\.24\.1\.
> 
> 
> \-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-\-
> 
> As reported at http&#8203;://bugs\.debian\.org/915366&#8203;:
> 
> "perlfaq4 has a question "Why don't my \<\<HERE documents work?"\, and
> much
> of the answer deals with ways to write indented here documents\. These
> are largely kludges\, however\, and the faq should really mention the ~
> modifier \(desribed in perlop\)\, which was introduced precisely in order
> to provide a clean way to do this\."
> 
> The author suggests an addendum&#8203;:
> 
> "Beginning with Perl version 5\.26\, a much simpler and cleaner way to
> write indented here documents has been added to the language&#8203;: the
> tilde
> \(~\) modifier\. See "Indented Here\-docs" in perlop for details\."
> 
> 

This looks good to me\, but perlfaq is maintained upstream on CPAN\, so that's where the wording will have to be revised first\.

Can you open a ticket at&#8203;:

https://github.com/perl-doc-cats/perlfaq/issues

Thank you very much\.

\-\- 
James E Keenan \(jkeenan@&#8203;cpan\.org\)

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @jmdh

On Tue\, Jan 01\, 2019 at 07&#8203;:32&#8203;:49AM \-0800\, James E Keenan via RT wrote&#8203;:
> This looks good to me\, but perlfaq is maintained upstream on CPAN\, so that's where the wording will have to be revised first\.
> 
> Can you open a ticket at&#8203;:
> 
> https://github.com/perl-doc-cats/perlfaq/issues

PR at https://github.com/perl-doc-cats/perlfaq/pull/77

**Comment by p5pRT** [comment]
### From @tonycoz

On Tue\, 01 Jan 2019 08&#8203;:51&#8203;:32 \-0800\, dom wrote&#8203;:
> On Tue\, Jan 01\, 2019 at 07&#8203;:32&#8203;:49AM \-0800\, James E Keenan via RT wrote&#8203;:
> > This looks good to me\, but perlfaq is maintained upstream on CPAN\, so
> > that's where the wording will have to be revised first\.
> >
> > Can you open a ticket at&#8203;:
> >
> > https://github.com/perl-doc-cats/perlfaq/issues
> 
> PR at https://github.com/perl-doc-cats/perlfaq/pull/77

Thanks\, since it's been reported upstream\, closing here\.

Tony

**Comment by p5pRT** [comment]
@tonycoz - Status changed from 'open' to 'rejected'
